llmfoundry/models/layers/ffn.py

```diff
@@ -143,13 +171,17 @@ def build_ffn(
             expansion_ratio=expansion_ratio,
             fc_type=fc_type,
             ffn_hidden_size=ffn_hidden_size,
+            ffn_act_fn=ffn_act_fn,
             device=device,
             bias=bias,
         )
     elif ffn_type == 'te_ln_mlp':
         assert te is not None
         ffn_hidden_size = resolve_ffn_hidden_size(d_model, expansion_ratio,
                                                   ffn_hidden_size)
+        if ffn_act_fn is not None:
```

Comment:
super nit: do this before L180 to raise exception as early as possible

Reply:
added [here](https://github.com/mosaicml/llm-foundry/pull/805/files#diff-2f97d1b3cb814c319489ccdb2313215ee45e7f8d5ebeb57e381c5ea1353c05f4R298)